dirty_cat/super_vectorizer.py

```diff
@@ -24,29 +24,26 @@
 from dirty_cat import GapEncoder
 
 
+_sklearn_loose_version = LooseVersion(sklearn.__version__)
+
+
 def _has_missing_values(array: pd.DataFrame) -> bool:
     """
     Returns True if `array` contains missing values, False otherwise.
     """
     return any(array.isnull())
 
 
-def _replace_missing(array: pd.DataFrame) -> pd.DataFrame:
+def _replace_missing(array: pd.DataFrame, value: str = "missing") -> pd.DataFrame:
```

Comment:
I think that pep8 wants
```suggestion
def _replace_missing(array: pd.DataFrame, value: str="missing") -> pd.DataFrame:
```

Reply:
Also, to me it is surprising that a variable of type DataFrame is named "array". In this function, I would name it "df", as this is a typical convention for dataframes.

Reply:
When using this format `def func(argument: type=default_value)`, my IDE (PyCharm) raises warnings E251/E252.
See https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes

I found no written rule about that, nonetheless I believe the consensus is 
```
Don't use spaces around the '=' sign when used to indicate a keyword argument 
or a default parameter value, except when using a type annotation.
```

For the argument named `array`, you're right, I'll rename it `df`. It stayed that way because historically we also managed numpy arrays.